pkg/hacontroller/controller.go

```diff
@@ -190,7 +194,7 @@ type haController struct {
 }
 
 func (hac *haController) watchPods(ctx context.Context) (<-chan watch.Event, error) {
-	initialPods, err := hac.kubeClient.CoreV1().Pods("").List(ctx, hac.podListOpts)
+	initialPods, err := hac.kubeClient.CoreV1().Pods(metav1.NamespaceAll).List(ctx, hac.podListOpts)
```

Comment:
why did that change? is it a change or the same as `""`?

Reply:
Its a const `""`, so no real change

Reply:
as to why: I should have used it from the start and I felt a bit silly creating a PR with 3 changed `""`